core/unit_test/TestNumericTraits.hpp

```diff
@@ -210,9 +210,10 @@ TEST(TEST_CATEGORY, numeric_traits_infinity) {
 #endif
   TestNumericTraits<TEST_EXECSPACE, float, Infinity>();
   TestNumericTraits<TEST_EXECSPACE, double, Infinity>();
-  // FIXME_NVHPC long double not supported
-#if !defined(KOKKOS_ENABLE_CUDA) || \
-    !defined(KOKKOS_COMPILER_NVHPC)  // 23.7 long double
+  // FIXME_NVHPC long double not supported, 23.7 long double
```

Comment:
Not sure we can talk about a naming scheme but I notice that Christian was doing
`// FIXME_NVHPC <VERSION TESTED> <OPTIONAL DESCRIPTION>`

So on these I would suggest
```suggestion
  // FIXME_NVHPC 23.7 long double
```